Example/Example/CommonTableViewController.swift

```diff
@@ -165,7 +165,7 @@ extension CommonTableViewController: InputBarAccessoryViewDelegate {
         tableView.contentInset.bottom = size.height + 300 // keyboard size estimate
     }
     
-    func inputBar(_ inputBar: InputBarAccessoryView, textViewTextDidChangeTo text: String) {
+    @objc func inputBar(_ inputBar: InputBarAccessoryView, textViewTextDidChangeTo text: String) {
```

Comment:
Just curious why this? :)

Reply:
hmm. good question. I actually don't remember. I think it was an artifact of copying it from my app-code. iirc my class wasn't an NSObject subclass, so this was required.

just removed this and pushed 😄

Reply:
actually I take it back - looks like it is required. debugging now

Reply:
<img width="627" alt="Screen Shot 2021-03-05 at 4 42 43 PM" src="https://user-images.githubusercontent.com/3814772/110188798-d0d21d80-7dd1-11eb-8e06-14de6483bbdc.png">

looks like `ButtonAnimationExample.swift` needs it to be marked @objc

Reply:
reverted back to include `@objc`